interview-guide-sre-ja.md

```diff
@@ -0,0 +1,129 @@
+# Quipper SRE Team 採用面接ガイド
+
+このドキュメントでは [Quipper SRE Team](https://career.quipper.com/jp/jobs/sre/) の採用プロセスにおける面接について説明します。
+
+## 目的
+
+Quipper の採用チームは「ミスマッチこそが採用活動における最大の失敗」と考えます。採用プロセスを通じてこれを避け、応募者と Quipper 双方にとって最良の結果を得ることを目指します。そのためには面接において以下が達成されるべきと考えています。
+
+- 面接において応募者に高いパフォーマンスを発揮してもらうこと
+- 面接がより効率的で有意義なものになること
+- 面接の場でお互いをより深く理解できること
+
+この内容は予告なく変更されることがありますが、より良い採用プロセスを実現するためのアップデートですのでご容赦ください。
+
+## 面接プロセス共通の注意事項
+
+現在は全面オンライン実施となります（カメラはオンにしていただいていますが、服装はリラックスできる格好で構いません）
+
+履歴書・職務経歴書の持参は不要です。事前に提出いただいたドキュメントは面接官に渡されています。
+
+面接中、PCやノートでメモを取っていただいても構いません。
+
+面接・面談は日本語で行われます。
+
+## 一次面接
+
+### 出席者
+
+応募者1名に対し、Quipper SRE Team の Engineering Manager と Member の合計3名が面接官として参加いたします。
+
+面接官のうち1名がファシリテーションを行いますが、適宜他の面接官もフォローや質問を行います。
+
+### 目的
+
+候補者が Quipper SRE Team で働くことが双方にとって良いことか、そのマッチングを見極めます。
+
+相互に理解することを一番の目的としているので、面接中のどのタイミングにおいても候補者から質問していただけます。
+
+面接は日本語で行われます。
+
+### 時間割
+
+一次面接は以下の流れで進行します。
+
+1. 面接官の自己紹介
+2. 候補者の自己紹介
+3. 定型的な質問
+    - 転職活動の状況 (そもそも考えているか・しているか)
+    - 転職先に求めるもの (業務、待遇、文化、事業、等)
+    - Quipper に興味を持ったポイント・志望理由
+4. 以降、候補者・面接官からの質問と回答
+
+---
```

Comment:
I think this line `---` isn't needed.

Reply:
Addressed cd841ff09971a7319c2876fe3431898b87c5a89c